federatedscope/cl/loss/NT_xentloss.py

```diff
@@ -0,0 +1,41 @@
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+
+from federatedscope.register import register_criterion
+
+class NT_xentloss(nn.Module):
+    def __init__(self, temperature=0.5):
+        super(NT_xentloss, self).__init__()
+        self.temperature = temperature
+        
+    def forward(self, z1, z2):
+        N, Z = z1.shape 
```

Comment:
copyright

Reply:
docstring to describe  the input tensor's shape

Reply:
finish